package.json

```diff
@@ -34,16 +36,18 @@
     "neutrino": "9.5.0",
     "ol": "7.3.0",
     "prettier": "2.8.7",
+    "shx": "0.3.4",
     "standard-version": "9.5.0",
     "start-server-and-test": "1.11.5",
     "stylelint": "15.4.0",
     "stylelint-config-standard": "32.0.0",
+    "typescript": "5.0.4",
     "webpack": "4.44.2",
     "webpack-cli": "3.3.12",
     "webpack-dev-server": "3.11.0"
   },
   "scripts": {
-    "build": "webpack --mode production ",
+    "build": "shx rm -rf build && tsc --project config/tsconfig-build.json && esbuild build/index.js --bundle --outfile=build/bundle.js --loader:.svg=text",
```

Comment:
Is there multiple tsconfig ? if no you can simply call it tsconfig.json and remove the --project parameter

Reply:
Not yet, but I guess we should have a default tsconfig.json file for development pretty soon. That's how OL is doing it